script.js

```diff
@@ -0,0 +1,68 @@
+let content;
+window.onload = function() {content = document.getElementById("content");}
+const doneTaskTextStyle = "line-through";
+
+function addEmptyTask(){
+    let task = createEmptyTask();
+    content.appendChild(task);
+}
+
+function createRawTask(){
+    // this function creates just the task object with the relevent event handlers, without inner elements
+    let newTask = document.createElement("div");
+    newTask.classList.add("task")
+    return newTask;
+}
+
+function addTaskInputBar(task){
+    // this function will add to the task the input bar section, and will return the input bar object.
+    let taskInput = document.createElement("input");
+    taskInput.classList.add("taskInputText");
+    taskInput.addEventListener("keydown",(event) => {
+        if (event.key == "Enter"){
```

Comment:
read about the difference between `==` and `===`

Reply:
done